perf/lua/1mops_write.lua

```diff
@@ -20,12 +20,16 @@ local params = require('internal.argparse').parse(arg, {
     {'nodes', 'number'},
```

Comment:
commit message probably should be ` perf: standardize an output of 1mops_write benchmark `

Reply:
In that case, the commit title is more than 50 symbols, so I shorten it to this one.